packages/salesforcedx-vscode-core/src/commands/listMetadata.ts

```diff
@@ -35,9 +35,9 @@ export class ForceListMetadataExecutor extends SfdxCommandletExecutor<string> {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   public build(data: {}): Command {
     const builder = new SfdxCommandBuilder()
-      .withArg('force:mdapi:listmetadata')
+      .withArg('org:list:metadata')
       .withFlag('-m', this.metadataType)
-      .withFlag('-u', this.defaultUsernameOrAlias)
+      .withFlag('-o', this.defaultUsernameOrAlias)
       .withLogName('force_mdapi_listmetadata')
```

Comment:
```suggestion
      .withLogName('list_metadata')
```

Reply:
Are we changing telemetry names in this pass?